docs/react-jss.md

```diff
@@ -587,4 +587,121 @@ const Button = () => {
 
 ## TypeScript
 
-TODO hooks support
```

Comment:
Lets move changes to react-jss-ts.md? So that we have space for all TS specific things and link that file from react-jss.md and other places where we find the right fit. Good places to link is super important for discoverability!